content/team/index.md

```diff
@@ -0,0 +1,258 @@
+---
+pagetype: major
+pagename: team
+section: Team
+title: Team | IPFS
+url: team
+save_as: team/index.html
+constellation: constellation-02.svg
+---
+
+The IPFS Team is a collection of thousands of contributors from multiple geographies and backgrounds, interested in making the Web Distributed.
+
+```
+TODO Insert nice animation with World Map and hex faces
+```
+
+To Work together collaboratively, distributed and asynchronously, [we've developed multiple habits for Synchronous and Asynchronous communication and teams to focus on different areas of focus](https://github.com/ipfs/team-mgmt#how-we-work-together). In this page, you can learn about the multiple IPFS Working Groups and how to join the conversation.
+
+## IPFS Working Groups
+
+The IPFS Working Groups are teams of people that are appointed to research, develop, and deploy work under the working groups' scope. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+You can learn more about this group structure at [TEAM_ROLES_STRUCTURES.md](https://github.com/ipfs/team-mgmt/blob/master/TEAMS_ROLES_STRUCTURES.md#working-groups) in the ipfs/team-mgmt repo.
+
+The active IPFS Working Groups list:
+
+- [**Project**](#project)
```

Comment:
I'm not sure this list reflects our most recent capacity and prioritization discussions. Perhaps wait to publish this part for a few more weeks, until we can finish that convo?